README.md

```diff
@@ -42,11 +42,13 @@ human-readable size or throughput. It is localized to:
 
 [https://python-humanize.readthedocs.io](https://python-humanize.readthedocs.io)
 
+<!-- usage-start -->
+
 ## Usage
 
 ### Integer humanization
 
-```pycon
+```python
```

Comment:
Let's keep `pycon` for these with Python console output:

<img width="809" alt="image" src="https://user-images.githubusercontent.com/1324225/155883429-c2903baa-eae6-4552-b1e8-7f4e94f104b6.png">

https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer